dds_cli/__main__.py

```diff
@@ -1243,6 +1243,40 @@ def delete_project(click_ctx, project: str):
         sys.exit(1)
 
 
+# -- dds project status extend -- #
+@project_status.command(name="extend", no_args_is_help=True)
+# Options
+@project_option(required=True)
+@click.option(
+    "--new_deadline",
```

Comment:
Personal preference and to keep it consistent with other options. The argument in the `def` line is still kept as `new_deadline`
```suggestion
    "--new-deadline",
```